test/TesterInternal/General/RequestContextTest.cs

```diff
@@ -177,6 +176,7 @@ public void Dispose()
             RequestContext.Clear();
         }
 
+#if !NETSTANDARD
```

Comment:
Because if you notice CM in the test name, it was actually testing whether the correlation manager's ActivityId is propagated implicitly, which is not supported in net standard